ci-operator/step-registry/windows/e2e/operator/test/upgrade/upgrade/windows-e2e-operator-test-upgrade-upgrade-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export WMCO_NS=openshift-windows-machine-config-operator
```

Comment:
This should be defined at a higher level if possible.
At the chain or test level.

Reply:
Same for the use in ci-operator/step-registry/windows/e2e/operator/test/upgrade/install/windows-e2e-operator-test-upgrade-install-commands.sh